src/messages/index.ts

```diff
@@ -16,6 +16,7 @@ export const messages = {
   ngPresetMessage: 'Angular icons will be ',
   jsOfficialPresetMessage: 'Official JS icon will be',
   tsOfficialPresetMessage: 'Official TS icon will be',
+  jsonOfficialPresetMessage: 'Official JSON icon will be',
```

Comment:
@jens1o we're only providing a JSON toggle. If you wan to change the `php` icon you can do it by disabling `php` and [enabling `php2`](https://github.com/JimiC/vscode-icons/blob/cd9e9021949f44043228758af5167d6c487ffa7e/src/icon-manifest/supportedExtensions.ts#L210).